layouts/fastly/helix.vcl

```diff
@@ -1788,23 +1785,25 @@ sub hlx_bereq {
   if (req.http.X-Request-Type != "Preflight") {
     unset bereq.http.X-Orig-Url;
     unset bereq.http.X-Orig-Host;
+    unset bereq.http.X-Owner;
+    unset bereq.http.X-Repo;
+    unset bereq.http.X-Ref;
```

Comment:
```suggestion
```

@tripodsan  Potentially. I'd leave the headers in so we don't break anything.

Reply:
?? I just moved the `unset`s into the if. they were removed before.

Reply:
Nevermind then.